master/static/srv/entrypoint.sh

```diff
@@ -3,12 +3,31 @@
 set -e
 set -x
 
+STDOUT_FILE=/run/determined/train/logs/stdout.log
+STDERR_FILE=/run/determined/train/logs/stderr.log
+
+mkdir -p "$(dirname "$STDOUT_FILE")" "$(dirname "$STDERR_FILE")"
+
 # Create symbolic links from well-known files to this process's STDOUT and
 # STDERR. Anything written to those files will be inserted into the output
 # streams of this process, allowing distributed training logs to route through
 # individual containers rather than all going through SSH back to agent 0.
-ln -sf /proc/$$/fd/1 /run/determined/train/stdout.log
-ln -sf /proc/$$/fd/2 /run/determined/train/stderr.log
+ln -sf /proc/$$/fd/1 "$STDOUT_FILE"
+ln -sf /proc/$$/fd/2 "$STDERR_FILE"
+
+if [ -n "$DET_K8S_LOG_TO_FILE" ]; then
+    # TODO: Do this in the base environment image.
+    apt-get update && apt-get install -y daemontools
```

Comment:
For airgapped k8s clusters (maybe there is one 🤷), we definitely should move this before landing.

Reply:
Oh yeah, this is totally temporary for the PR. Just didn't feel like waiting on a cycle with the environments repo to get the first version out, though I guess I can go ahead and get that started at any point.